ingestion-beam/src/test/resources/schemaAliasing/example-aliasing-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "aliases": [
+    {
+      "alias": "some-product/baseline",
+      "base": "glean/baseline"
+    },
```

Comment:
@fbertsch how do you like this way of configuring aliases?

Reply:
I thought we would want to alias entire namespaces, so we could have something like 
```
{
  "alias": "some-product",
  "base": "glean"
}
```

without having to specify each individual `docType`.  This would mean `some-product/*` would map to `glean/*`.

However, re-reading #424 seems to match the approach here.

An advantage to aliasing an entire namespace would be that when Glean adds a new doc type, all applications would be able to use it without adding more aliases. A disadvantage would be that we still need to manage the single glean schema being used by multiple doctypes in the mozilla-pipeline-schemas repo and build step.

Reply:
I prefer this approach. The probe-scraper will keep track of default pings and new pings added per-product, so we won't have to take care of that in mozilla-pipeline-schemas, which seems appropriate since we will also keep track of new repositories in the probe-scraper. That way all of the information is contained to one place.